IdentityCore/src/controllers/MSIDBaseRequestController.h

```diff
@@ -44,8 +45,12 @@ typedef void(^MSIDAuthorityCompletion)(BOOL resolved, NSError * _Nullable error)
                                 fallbackController:(nullable id<MSIDRequestControlling>)fallbackController
                                              error:(NSError * _Nullable * _Nullable)error;
 
+#if !EXCLUDE_FROM_MSALCPP
 - (nullable MSIDTelemetryAPIEvent *)telemetryAPIEvent;
 - (void)stopTelemetryEvent:(nonnull MSIDTelemetryAPIEvent *)event error:(nullable NSError *)error;
+#endif
+
+#define CONDITIONAL_STOP_TELEMETRY_EVENT(x, y) CONDITIONAL_COMPILE_MSAL_CPP([self stopTelemetryEvent:(x) error:(y)])
```

Comment:
Should this be moved to MSIDTelemetryConditionalCompile.h?

Reply:
I still prefer to keep it here, since the caller is calling [stopTelemetryEvent], and this MACRO is defined where [stopTelemetryEvent] is declared. It would be more convenient for the person who read this code in the future.

Reply:
Okay, could you at least move this to the top of the class? It looks completely out of place here....